src/uwtools/tests/fixtures/rocoto_invalid.xml

```diff
@@ -0,0 +1,43 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE workflow [
+  <!ENTITY ACCOUNT "myaccount">
+  <!ENTITY FOO "test.log">
+
+]>
+<workflow realtime="False" scheduler="slurm" >
+
```

Comment:
Here I removed the `cycledef` tag that is required to return true from the Rocoto schema validation. Removing the `cycledef` tag will cause the validation to fail to confirm our unit tests are catching this appropriately. If helpful I can make this more obvious by changing the file name as needed to help highlight this better.
i.e. `invalid_rocoto_removed_cycledef.xml`

Line removed: `<cycledef group="howdy">202209290000 202209300000 06:00:00</cycledef>`

Reply:
Seems like a good approach to me, and I like the filename. It might be good to add an XML comment saying what's going on.